boot.php

```diff
@@ -2344,6 +2322,7 @@ function curPageURL() {
 function random_digits($digits) {
 	$rn = '';
 	for ($i = 0; $i < $digits; $i++) {
+		/// @TODO rand() is different to mt_rand() and maybe lesser "random"
```

Comment:
In practice this isn't used for sensible usage like cryptography, so a strong entropy isn't crucial.